fe/src/components/Pagination/PaginationBase.tsx

```diff
@@ -60,37 +63,57 @@ export const PaginationBase: React.FC = () => {
           </p>
         </div>
         ) : (
-            <>
-            <div className="items-count">
-              <div className="items-count__name">Items on page</div>
-              <div>
-                <select
-                  className="items-count__select"
-                  value={itemsPerPage}
-                  onChange={handleItemsPerPage}
-                >
-                  <option value="4">4</option>
-                  <option value="8">8</option>
-                  <option value="12">12</option>
-                  <option value="16">16</option>
-                </select>
+          <>
+            <div className="container">
+              <Breadcrumbs />
+              <h1 className="page-title">Mobile Phones</h1>
+              <p className="page-total-models">{products.length} models</p>
+              <div className="sort-block">
+                <div className="items-count">
+                  <div className="items-count__name">Sort by</div>
+                  <div>
+                    <select
+                      className="items-count__select"
+                      value={itemsPerPage}
+                      onChange={handleItemsPerPage}
+                    >
+                      <option value="71">Newest</option>
+                      <option value="16">Cheap first</option>
+                      <option value="12">Expensive first</option>
+                    </select>
+                  </div>
+                </div>
+
+                <div className="items-count">
+                  <div className="items-count__name">Items on page</div>
+                  <div>
+                    <select
+                      className="items-count__select"
+                      value={itemsPerPage}
+                      onChange={handleItemsPerPage}
+                    >
+                      <option value="71">All</option>
```

Comment:
```suggestion
                      <option value="71">All</option>
```
let's get rid of the 'All' option. Could we have only a fixed amount of products per page?